### PR TITLE
make a withdrawl 

### PR DESCRIPTION
**Pull request by rwobben**

### CODE REVIEW

**Comment by rwobben** [comment]
Solved #20

File: paytime/src/Lib.hs [hunk 1]
```diff
@@ -93,8 +93,36 @@ doDepositAccounts accts = do
...
+         descriptionWithDrawl <- getLine
+         return $ withDrawlToAccountInStore (read amountToWithDrawl::Integer) descriptionWithDrawl accountToWitdrawl accts
+    else do
+         putStrLn "Amount cannot be negative"
+         return accts
+  else do 
+    putStrLn "This account does not exist"
+    return accts  
+
+withDrawlToAccountInStore ::  Integer -> String -> String -> AccountStore -> AccountStore
```
Comment (GetContented) [inline paytime/src/Lib.hs]: You don't withdraw To an account, you withdraw From an account. The verb is withdraw, not withdrawl

File: paytime/app/Main.hs [hunk 2]
```diff
@@ -15,7 +15,7 @@ main = runLoop Map.empty
           putStrLn "Menu."
           putStrLn $ "CA: Create Account." ++
             (if canDelete then "\nDA: Delete Account." else "")
-            ++ "\nSA: Show Accounts. \nMD: Make Deposit. \n Q: Quit"
+            ++ "\nSA: Show Accounts. \nMD: Make Deposit. \nMW: Make WithDrawl   \n Q: Quit"
```
Comment (GetContented) [inline paytime/app/Main.hs]: It's spelt withdrawal, not withdrawl.

File: paytime/src/Lib.hs [hunk 3]
```diff
@@ -93,8 +93,36 @@ doDepositAccounts accts = do
...
+-- making a withdrawl.
+
+doWithDrawlAccounts :: AccountStore -> IO AccountStore
+doWithDrawlAccounts accts = do
+  putStrLn "To which account do you want to withdraw? "
+  accountToWitdrawl <- getLine
+  if doesAccountExistInAccounts accountToWitdrawl accts
+  then do
+    putStrLn "Which amount do you want to deposit?"
+    amountToWithDrawl <- getLine
```
Comment (GetContented) [inline paytime/src/Lib.hs]: this is a spelling mistake. It should be

amountToWithdraw

Comment (GetContented) [inline paytime/src/Lib.hs]: Also, your message to the user is incorrect. It currently asks about deposit.

File: paytime/src/Lib.hs [hunk 4]
```diff
@@ -93,8 +93,36 @@ doDepositAccounts accts = do
...
-depositToAccountInStore ::  Integer -> String -> String -> AccountStore -> AccountStore
-depositToAccountInStore amount description = Map.adjust (deposit amount description)
+depositToAccountInStore ::  Integer -> String -> String -> AccountStore -> AccountStore 
+depositToAccountInStore amount description = Map.adjust (deposit amount description) 
+
+-- making a withdrawl.
+
+doWithDrawlAccounts :: AccountStore -> IO AccountStore
+doWithDrawlAccounts accts = do
+  putStrLn "To which account do you want to withdraw? "
```
Comment (GetContented) [inline paytime/src/Lib.hs]: Shoulb be "from", not "to".

File: paytime/app/Main.hs (hunk 2 above)
Comment (rwobben) [inline paytime/app/Main.hs]: will change it. Difficult as English is not my mother language

File: paytime/src/Lib.hs (hunk 3 above)
Comment (rwobben) [inline paytime/src/Lib.hs]: Will also change that

File: paytime/app/Main.hs (hunk 2 above)
Comment (GetContented) [inline paytime/app/Main.hs]: Yeah, I know. I think that's why sometimes our communication is difficult.

I'm not sure if you want me to tell you these things or not.

File: paytime/src/Lib.hs (hunk 1 above)
Comment (rwobben) [inline paytime/src/Lib.hs]: Will also be chancing that.  Thanks for the feedback

File: paytime/app/Main.hs (hunk 2 above)
Comment (rwobben) [inline paytime/app/Main.hs]: yes. I want it. I want to make good programms.

**Comment by rwobben** [comment]
I hope I solved them all. 
Otherwise point me to them or make a PR for it

**Comment by GetContented** [comment]
haha it's fine.

**Comment by rwobben** [comment]
Oke, then I will commit it.

**Pull request merged**